plugins/inputs/syslog/syslog.go

```diff
@@ -30,13 +30,14 @@ var sampleConfig string
 const readTimeoutMsg = "Read timeout set! Connections, inactive for the set duration, will be closed!"
 
 type Syslog struct {
-	Address        string                     `toml:"server"`
-	Framing        string                     `toml:"framing"`
-	SyslogStandard string                     `toml:"syslog_standard"`
-	Trailer        nontransparent.TrailerType `toml:"trailer"`
-	BestEffort     bool                       `toml:"best_effort"`
-	Separator      string                     `toml:"sdparam_separator"`
-	Log            telegraf.Logger            `toml:"-"`
+	Address          string                     `toml:"server"`
+	Framing          string                     `toml:"framing"`
+	SyslogStandard   string                     `toml:"syslog_standard"`
+	Trailer          nontransparent.TrailerType `toml:"trailer"`
+	BestEffort       bool                       `toml:"best_effort"`
+	Separator        string                     `toml:"sdparam_separator"`
+	MaxMessageLength int                        `toml:"max_message_length"`
```

Comment:
Can we please make this a `config.Size` so people can use `8kB` as size values i.e. use more human-readable units?